### PR TITLE
add code to convert unix date on events page to human readable date

### PR DESCRIPTION
**Pull request by mogannam**

### CODE REVIEW

**Review by robyng — DISMISSED** [review]
Thanks

**Comment by robyng** [comment]
looks good but we don't need seconds listed on timestamp

File: client/src/components/EventCard/index.js [hunk 1]
```diff
@@ -22,6 +22,21 @@ AWS.config.update({
...
+  if(event.when){
+    const dateObj = new Date(parseInt(event.when));
+    formatDate = dateObj
+    formatDate = dateObj.toLocaleDateString('en-US', {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+      hour: 'numeric',
+      minute: 'numeric',
+      second: 'numeric',
```
Comment (robyng) [inline client/src/components/EventCard/index.js]: comment out seconds for timestamp.

**Comment by mogannam** [comment]
Removed seconds

**Review by robyng — APPROVED** [review]

**Pull request merged**